src/settings/index.scss

```diff
@@ -107,22 +107,30 @@ body {
 	/* Add new source menu */
 	.components-menu-group {
 
-		div {
-			display: flex;
+		> div {
+			display: grid;
+			grid-template-columns: repeat(3, 1fr);
 			width: 368px;
 			align-items: flex-start;
 			align-content: flex-start;
 			gap: 8px;
 			padding: 8px 8px 16px 8px;
-			flex-wrap: wrap;
+
+			.rdb-settings-page_add-data-source-btn-wrapper {
+				height: 100%;
+			}
```

Comment:
Is this needed? I'm not seeing any change with or without it

Reply:
Without it I'm seeing it not fill the full height of the row which means the hover size is different for each item.

![image](https://github.com/user-attachments/assets/c75d8c32-4a59-433a-85bf-82860a2aa523)

Reply:
Thanks for the screenshot! I see this now using FF. In Chrome, it appears differently 🤷‍♀️